docs/global-styles.md

```diff
@@ -0,0 +1,20 @@
+# Global Styles
```

Comment:
I think this documentation is most likely to be useful if you're using the legacy stuff, with this in mind should we align this work with the compatibility mode stuff?

User scenario:
1. I have GOV.UK Template, and just want a build that works with that

Reply:
in that case you don't need global styles then, right? As they're coming from template?

Reply:
all of the *compatibillity vars are set to true in Frontend 🤔